about-blockcore.md

```diff
@@ -0,0 +1,28 @@
+**What is Blockcore?**
```

Comment:
Duplicate file and text, should we keep this?

Reply:
No, I will remove this ... I started with this and there was a suggestion to move into the main readme.